### PR TITLE
Update microsoft-staffhub-to-be-retired.md

### PR DESCRIPTION
**Pull request by v-behow**
Updating Office 365 to Microsoft 365

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/expand-teams-across-your-org/shifts/microsoft-staffhub-to-be-retired.md [hunk 1]
```diff
@@ -122,17 +122,17 @@ By default, Teams is enabled for all organizations. You assign Teams licenses to
...
 
-Yes, each manager and team member using Teams and Shifts within Teams must have an Office 365 account (Azure AD) and must be assigned a valid license.
+Yes, each manager and team member using Teams and Shifts within Teams must have a Microsoft 365 account (Azure AD) and must be assigned a valid license.
  
-#### In StaffHub, I could add a special prefix to all Office 365 Groups that were created by StaffHub teams. Can I do the same in Teams?
+#### In StaffHub, I could add a special prefix to all Microsoft 365 Groups that were created by StaffHub teams. Can I do the same in Teams?
 
-You won't have the same capability when you move from StaffHub to Teams. If you need to apply name consistency to all your Office 365 Groups, you can use [Office 365 Groups naming policy](https://docs.microsoft.com/office365/admin/create-groups/groups-naming-policy).
+You won't have the same capability when you move from StaffHub to Teams. If you need to apply name consistency to all your Office 365 Groups, you can use [Microsoft 365 Groups naming policy](https://docs.microsoft.com/office365/admin/create-groups/groups-naming-policy).
```
Comment (JohanFreelancer9) [inline Teams/expand-teams-across-your-org/shifts/microsoft-staffhub-to-be-retired.md:131]:
```suggestion
You won't have the same capability when you move from StaffHub to Teams. If you need to apply name consistency to all your Microsoft 365 Groups, you can use [Microsoft 365 Groups naming policy](https://docs.microsoft.com/office365/admin/create-groups/groups-naming-policy).
```

**Comment by scanum** [comment]
@v-behow Please sign the CLA. Thanks

**Comment by scanum** [comment]
@v-behow Please sign the CLA. Thanks

**Comment by scanum** [comment]
@v-behow We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this PR.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**